packages/nx/src/command-line/connect/view-logs.ts

```diff
@@ -7,8 +7,8 @@ import { readNxJson } from '../../config/nx-json';
 import { connectExistingRepoToNxCloudPrompt } from './connect-to-nx-cloud';
 
 export async function viewLogs(): Promise<number> {
-  const cloudUsed = isNxCloudUsed(readNxJson());
-  if (cloudUsed) {
+  const [isCloudUsed, _] = isNxCloudUsed(readNxJson());
```

Comment:
The second parameter is not used so there is no need to define it.

```suggestion
  const [isCloudUsed] = isNxCloudUsed(readNxJson());
```